pkg/intermediate/aggregate.go

```diff
@@ -331,6 +340,14 @@ func (a *AggregationProcess) addOrUpdateRecordInMap(flowKey *FlowKey, record ent
 		}
 		if !correlationRequired {
 			aggregationRecord.ReadyToSend = true
+			// if no correlation is required for inter-Node record, K8s metadata will not
```

Comment:
"If no correlation is required for an inter-Node record, K8s metadata is expected to be not completely filled."